src/http.ts

```diff
@@ -133,16 +133,20 @@ export async function getJSON<T>(
   }
 
   const {
-    json: { error, error_description, ...success },
+    json: { error, error_description, ...data },
```

Comment:
Renamed this to data, because it contains the mfa_token and it was a bit weird to use `success` in case of `an error`.